sources/packages/backend/apps/api/src/route-controllers/overaward/_tests_/e2e/overaward.institution.controller.getOverawardsByStudent.e2e-spec.ts

```diff
@@ -0,0 +1,108 @@
+import { HttpStatus, INestApplication } from "@nestjs/common";
+import * as request from "supertest";
+import {
+  createFakeStudent,
+  createFakeDisbursementOveraward,
+  createFakeUser,
+  createFakeStudentAssessment,
+  createFakeApplication,
+} from "@sims/test-utils";
+import { Repository } from "typeorm";
+import {
+  Application,
+  AssessmentTriggerType,
+  DisbursementOveraward,
+  DisbursementOverawardOriginType,
+  Student,
+  StudentAssessment,
+  User,
+} from "@sims/sims-db";
+
+import {
+  BEARER_AUTH_TYPE,
+  createTestingAppModule,
+  getInstitutionToken,
+  InstitutionTokenTypes,
+} from "../../../../testHelpers";
+import { OverawardDetailsAPIOutDTO } from "../../models/overaward.dto";
+import { getUserFullName } from "../../../../utilities";
+
+describe("OverawardInstitutionController(e2e)-getOverawardsByStudent", () => {
+  let app: INestApplication;
+  let userRepo: Repository<User>;
+  let studentRepo: Repository<Student>;
+  let assessmentRepo: Repository<StudentAssessment>;
+  let applicationRepo: Repository<Application>;
+  let disbursementOverawardRepo: Repository<DisbursementOveraward>;
+
+  beforeAll(async () => {
+    const { nestApplication, dataSource } = await createTestingAppModule();
+    app = nestApplication;
+    userRepo = dataSource.getRepository(User);
+    studentRepo = dataSource.getRepository(Student);
+    assessmentRepo = dataSource.getRepository(StudentAssessment);
+    applicationRepo = dataSource.getRepository(Application);
+    disbursementOverawardRepo = dataSource.getRepository(DisbursementOveraward);
+  });
+
+  it("Should return student overawards", async () => {
+    // Arrange
+    const user = await userRepo.save(createFakeUser());
```

Comment:
is it possible to simplify the "Arrange" area using the `saveFakeApplication`?

Reply:
Unless I am missing something, student and user will also be created by `saveFakeApplication` so we would not need the 49 and 50.